tests/test_other.py

```diff
@@ -8103,6 +8103,39 @@ def test_no_legalize_js_ffi(self):
       assert i_legalimport_i64, 'legal import not generated for invoke call'
       assert e_legalstub_i32, 'legal stub not generated for dyncall'
 
+  def test_export_aliasee(self):
+    # test aliasee is exported when alias is exported
+    if self.is_wasm_backend():
+      self.skipTest('not testing export aliasee and wasm backend')
+    wasm_dis = os.path.join(Building.get_binaryen_bin(), 'wasm-dis')
+
+    with env_modify({'EMCC_FORCE_STDLIBS': 'libc++'}):
+      # build side module
+      args = ['-s', 'EXPORT_ALL=0', '-s', 'SIDE_MODULE=1', '-O3', '-s', 'DISABLE_EXCEPTION_CATCHING=0']
+      print(args)
+      cmd = [PYTHON, EMCC, path_from_root('tests', 'other', 'alias', 'side.cpp'), '-g', '-o', 'side.wasm'] + args
+      print(' '.join(cmd))
+      run_process(cmd)
+
+      # build main module
+      args = ['-s', 'EXPORT_ALL=0', '-s', 'EXPORTED_FUNCTIONS=["_main", "_wprintf","__ZTVSt12length_error","__ZNSt12length_errorD1Ev","__ZNSt11logic_errorC2EPKc"]', '-s', 'MAIN_MODULE=2', '-O3', '-s', 'DISABLE_EXCEPTION_CATCHING=0']
+      cmd = [PYTHON, EMCC, path_from_root('tests', 'other', 'alias', 'main.cpp'), '-g', '-o', 'main.out.js'] + args
+      print(' '.join(cmd))
+      run_process(cmd)
+      run_process([wasm_dis, 'main.out.wasm', '-o', 'main.out.wast'])
+      text = open('main.out.wast').read()
+      # remove internal comments and extra whitespace
+      text = re.sub(r'\(;[^;]+;\)', '', text)
+      text = re.sub(r'\$var\$*.', '', text)
+      text = re.sub(r'param \$\d+', 'param ', text)
+      text = re.sub(r' +', ' ', text)
+      # print("text: %s" % text)
```

Comment:
can we remove all the text parsing here? if it is enough to just test for success in running the program, that is simpler and better.

Reply:
sure let me do that.